File: hw22/sqrts.py
# print square roots in Python language.  R. Brown, 9/2010 

import sys
from math import sqrt
if len(sys.argv) < 2:
    print("Enter in a set of space separated numbers so the program can set largest one to n, and caculate the all the square roots of 1 - n\n")
else:
    maxval = int(sys.argv[1])
    for count, data in enumerate(sys.argv):
        if count < 2:
            continue
        current = int(data)
        maxval = max(maxval, int(data))

    print("sqrt(n)\n--------")

    for n in range(1, maxval+1):
        print(sqrt(n))

File: hw22/Sqrts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Sqrts {
  public static void main(String args[]) {
    int n;
    if(Integer.parseInt(args[0]) < 1) {
        System.out.println("Please provide a valid value");
    } else {
        for (n=0;  n<=Integer.parseInt(args[0]);  n++)
            System.out.println("sqrt(n)");
            System.out.println("--------");
            System.out.println(Math.sqrt(n));
        return;        
    }
  }
}